NuCivic/dkan
# Workflow 508 compliance - CIVIC-4740

Issue: CIVIC-4740

## Description

Empty form labels on checkboxes. Users 'My Workbench' pages list VBO items with checkboxes, these checkboxes need labels.


## How to reproduce

1. Enable dkan_workflow
2. Create a dataset and resource, visit the My Workbench > My Drafts page
3. Enable the Chrome WAVE extension and confirm that there are red flagged errors on the checkboxes.


## QA Tests

- [x] Enable dkan_workflow
- [x] Create a dataset and resource, visit the My Workbench > My Drafts page
- [x] Enable the Chrome WAVE extension and confirm that there are NO red flagged errors on the checkboxes.
https://chrome.google.com/webstore/detail/accessibility-developer-t/fpkknkljclfencbdbgkenhalefipecmb?hl=en
- [x] Update the items to the 'Needs Review' status and check them on the Needs Review page, enable the WAVE extension, confirm NO red flags
